### PR TITLE
Update nebula-publishing-plugin to 19.2.0

### PR DESCRIPTION
**Pull request by reta**
Signed-off-by: Andriy Redko <andriy.redko@aiven.io>

### Description
Update nebula-publishing-plugin to 19.2.0

### Issues Resolved
Closes https://github.com/opensearch-project/OpenSearch/issues/5702

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed per the DCO using --signoff
- [x] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by reta** [comment]
@tlfeng FYI :)

File: buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java [hunk 1]
```diff
@@ -108,8 +109,12 @@ public String call() throws Exception {
             // Here we manually add any project dependencies in the "shadow" configuration to our generated POM
             publication.getPom().withXml(xml -> {
                 Node root = xml.asNode();
-                root.appendNode("name", project.getName());
-                root.appendNode("description", project.getDescription());
+                if (((NodeList) root.get("name")).isEmpty()) {
```
Comment (reta) [inline buildSrc/src/main/java/org/opensearch/gradle/PublishPlugin.java:112]: May conflict with `com.netflix.nebula.maven-base-publish` which adds `name` and `description` properties

**Comment by mch2** [comment]
> 

Muting this SR test - https://github.com/opensearch-project/OpenSearch/pull/5714

**Review by mch2 — APPROVED** [review]

**Review by dreamer-89 — APPROVED** [review]

**Comment by ashking94** [comment]
Test failure: #5765

**Pull request merged**